ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -322,8 +322,14 @@ instance (ShelleyCompatible proto era, ProtoCrypto proto ~ crypto) => QueryLedge
 
           case mPoolIds of
             Nothing ->
+              let poolIds = Set.fromList $ mconcat
+                    [ VMap.elems (SL._delegations _pstakeMark)
+                    , VMap.elems (SL._delegations _pstakeSet)
+                    , VMap.elems (SL._delegations _pstakeGo)
+                    ]
+              in
               StakeSnapshots
-                { ssStakeSnapshots = getPoolStakes (Set.fromList (VMap.elems (SL._delegations _pstakeMark)))
+                { ssStakeSnapshots = getPoolStakes poolIds
```

Comment:
Hmm, we changed the returned values in `ssStakeSnapshots`. Shouldn't we update the haddocks for this field?

Reply:
What comes back in that field depends on what is queried for.